app/components/application/FeedbackCard/FeedbackCard.tsx

```diff
@@ -13,36 +14,55 @@ export default function FeedbackBox({ className }: { className?: string }) {
   const {
     t: { feedback },
   } = useTranslate()
+  const [hovered, setHovered] = useState<boolean>(false)
+
+  const handleMouseEnter = () => {
+    setHovered(true)
+  }
+
+  const handleMouseLeave = () => {
+    setHovered(false)
+  }
 
   return (
-    <Card>
-      <div className="feedback box">
-        <div className="top">
-          <div className="row">
-            <div className="spaceholder">
-              <img
-                alt={feedback.feedbackShareAltText}
-                aria-hidden="true"
-                className="image"
-                src="/share-feedback.svg"
-              />
-            </div>
-            <div>
-              <h3 className="title">{feedback.feedbackTitle}</h3>
-              <p className="bodytext">{feedback.feedbackSubText}</p>
+    <a
+      aria-label="Click to share feedback with the Pokt Team in discord"
+      href="https://discord.gg/portal-rpc"
+      rel="noreferrer"
+      target="_blank"
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+    >
+      <Card
+        sx={
```

Comment:
Maybe we could get rid of `hovered` and avoid using the state here and replace it with:

```
sx={{
        '&:hover': {
          backgroundColor: "var(--mantine-color-navy-6)",
        },
      }}
```

Reply:
Good catch, I was trying to do this and totally forgot about "&". Thanks!